tests/plugins/widget/insertparagraphnexttowidget.js

```diff
@@ -23,6 +23,12 @@
 		bogusDiv = '<div>&nbsp;</div>';
 
 	bender.test( {
+		_should: {
+			ignore: {
+				// #4563
+				'test press shift + enter with ckeditor.enter_br inserts <br> element after a widget': bender.env.ie && bender.env.version == '8'
+			}
+		},
```

Comment:
I wonder what is the origin of this change here? :thinking:

Reply:
Well, we should ask @Dumluregn ;p
Commit related to this is described as  'Ignore test failing due to the other ticket.`.

Reply:
You got me, I've tried to smuggle it here...

Reply:
https://github.com/ckeditor/ckeditor4/issues/4563 - this is the ticket.